Jenkinsfile

```diff
@@ -200,8 +200,8 @@ pipeline {
           sh 'cd examples/nlp/token_classification && python punctuation_capitalization.py \
           model.data_dir=/home/TestData/nlp/token_classification_punctuation/ +pl.trainer.fast_dev_run=true \
           pl.trainer.gpus=2 pl.trainer.distributed_backend=ddp \
```

Comment:
Let's specify the index of the gpus:
```
pl.trainer.gpus=[0,1]
```

Reply:
@ericharper it would be create if pl.trainer.gpus=[1] or =[0] worked, then we can do parallel runs in CI

Reply:
for this model I'm leaving multi-gpu test